csharp/test/Drivers/Apache/Spark/NumericValueTests.cs

```diff
@@ -261,7 +262,8 @@ public async Task TestFloatValuesInsertSelectDelete(float value)
             string valueString = ConvertFloatToString(value);
             await InsertSingleValueAsync(table.TableName, columnName, valueString);
             object doubleValue = (double)value;
-            object floatValue = TestEnvironment.GetValueForProtocolVersion(doubleValue, value)!;
+            // Spark over HTTP returns float as double whereas Spark on Databricks returns float.
```

Comment:
Seems weird :/

Reply:
@CurtHagenlocher - Is it work converting FLOAT defined columns back to `float` type along with DATE/TIMESTAMP/DECIMAL?

Reply:
On MSSQL and many other relational databases, a 32-bit floating point value is REAL and a 64-bit floating point value is FLOAT. I don't know how it is with Spark. If a Spark FLOAT is a 32-bit floating point value then I think we should do the conversion from double -> float.

(This can obviously be a followup.)